codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -369,7 +371,6 @@ final DefaultPropertyKey verifyKey(PropertyKey key) {
         private short weight = DEFAULT_PRIORITY_WEIGHT;
         private DefaultStream parent;
         private IntObjectMap<DefaultStream> children = IntCollections.emptyMap();
-        private int prioritizableForTree = 1;
```

Comment:
Remind me what this was for again ... did a previous incarnation of the priority algorithm need it?